tools/half-sign.c

```diff
@@ -0,0 +1,117 @@
+#include <unistd.h>
```

Comment:
Explanation added. This isn't in use yet, but we're likely to add airgapped signatures from that code in the future.